# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1347545702 14400
#      Thu Sep 13 10:15:02 2012 -0400
# Node ID bff950876af5ae954751068629b6eec654a5aae1
# Parent  eec0d1fcba4f070e6de36dc56a3fb91a8ca04aeb
don't compiler kpse.cc by itself

* module.mk (UTIL_SRC): Remove util/kpse.cc from the list.

diff --git a/liboctave/util/module.mk b/liboctave/util/module.mk
--- a/liboctave/util/module.mk
+++ b/liboctave/util/module.mk
@@ -49,17 +49,16 @@ UTIL_C_SRC = \
   util/oct-rl-edit.c \
   util/oct-rl-hist.c 
 
 UTIL_SRC = \
   util/cmd-edit.cc \
   util/cmd-hist.cc \
   util/data-conv.cc \
   util/glob-match.cc \
-  util/kpse.cc \
   util/lo-array-gripes.cc \
   util/lo-ieee.cc \
   util/lo-utils.cc \
   util/oct-alloc.cc \
   util/oct-base64.cc \
   util/oct-glob.cc \
   util/oct-inttypes.cc \
   util/oct-locbuf.cc \
@@ -68,17 +67,17 @@ UTIL_SRC = \
   util/oct-shlib.cc \
   util/oct-sort.cc \
   util/pathsearch.cc \
   util/regexp.cc \
   util/singleton-cleanup.cc \
   util/sparse-sort.cc \
   util/sparse-util.cc \
   util/str-vec.cc \
-	$(UTIL_C_SRC)
+  $(UTIL_C_SRC)
 
 TEMPLATE_SRC += \
   util/oct-sort.cc
 
 OTHER_INC += \
   util/kpse.cc
 
 noinst_LTLIBRARIES += util/libutil.la
